packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -108,6 +108,10 @@ const ItemDisplay = ({
   if (icon) {
     displayIcon = <div className={cx('icon')}>{icon}</div>;
   }
+  let textWrapper = <span>{text}</span>;
+  if (textStyle === TextStyles.STRONG) {
+    textWrapper = <strong>{text}</strong>;
+  }
 
   let ariaLabel;
   if (textStyleMeaning) {
```

Comment:
These cases will also need to be updated to use `textWrapper` instead of `text` so that doesn't get lost when assigning an aria-label.

Reply:
Per offline discussion with @eawww, we do not need to use `textWrapper` to replace `text` for `ariaLabel`